# Message file for robots expression
# trial id
int64 id
# probability of happy
float64 p_happy
# probability of sad
float64 p_sad
# probability of neutral
float64 p_neutral
